### ISSUE TITLE
Update error messages

### ISSUE THREAD

**Issue by tsliaw** [hubmapconsortium/portal-ui#406]
Current error messages: 
400: Bad Request
Bad request: The HuBMAP API could not satisfy this request; If it should work, please file a bug.

401: Unauthorized
Unauthorized: Your credentials have expired.
Please login again.

401: Unauthorized
Unauthorized: You have not been added to the HuBMAP Group on Globus.
Request access here.

403: Access Denied
Access denied: For now, you need to login to access this resource. Eventually most resources will be available without login.

404: Not Found
Not found: If this page should exist, please file a bug.

500: Internal Service Error

504: Gateway Timeout
Gateway timeout: The HuBMAP API did not satisfy this request.
Either the API is down, or the server is running at an address which hasn't been whitelisted.

**Comment by mccalluc** [comment]
Part of the task is to determine what the new language should be?

**Comment by ngehlenborg** [comment]
Yes, that is the intention of this issue.

**Comment by ngehlenborg** [comment]
@mccalluc: are there indeed two different versions for 401s?

**Comment by mccalluc** [comment]
I believe this was fixed in #1044. Current messages (from `ErrorBody.jsx`):
```
  if (errorCode === 401 && isGlobus401) {
    return (
      <>
        Your credentials have expired. Please <LoginLink /> again.
      </>
    );
  }

  if (errorCode === 401) {
    return (
      <>
        You have not been added to the HuBMAP Group on Globus. Request access at <HelpEmailLink />.
      </>
    );
  }

  if (errorCode === 403 && isAuthenticated) {
    return (
      <>
        You may not have access to this resource. Request access at <HelpEmailLink />.
      </>
    );
  }

  if (errorCode === 403) {
    return (
      <>
        You may not have access to this resource or you need to <LoginLink />.
      </>
    );
  }

  if (errorCode === 404) {
    return (
      <>
        If this page should exist, submit a bug report to <HelpEmailLink />.
      </>
    );
  }

  return (
    <>
      If this problem persists, submit a bug report to <HelpEmailLink />.
    </>
  );
}
```
